### PR TITLE
[WORDPAD] Update Romanian (ro-RO) translation

### PR DESCRIPTION
**Pull request by AndreiMiloiu**
Attendum to https://github.com/reactos/reactos/pull/6517

I fixed the entire translation and all the accelerators in order to be everything as similar as in MSDN as possible.

### CODE REVIEW

**Comment by AndreiMiloiu** [comment]
Cc @reactos/lang-romanian

File: base/applications/wordpad/lang/ro-RO.rc [hunk 1]
```diff
@@ -32,31 +32,31 @@ BEGIN
     BEGIN
         MENUITEM "&Nou…\tCtrl+N",            ID_FILE_NEW
         MENUITEM "&Deschidere…\tCtrl+O",     ID_FILE_OPEN
-        MENUITEM "S&alvează\tCtrl+S",        ID_FILE_SAVE
-        MENUITEM "Salvea&ză ca…",            ID_FILE_SAVEAS
+        MENUITEM "S&alvare\tCtrl+S",        ID_FILE_SAVE
+        MENUITEM "Sal&vare ca…",            ID_FILE_SAVEAS
```
Comment (JoachimHenze) [inline base/applications/wordpad/lang/ro-RO.rc:36]:
```suggestion
        MENUITEM "S&alvare\tCtrl+S",         ID_FILE_SAVE
        MENUITEM "Sal&vare ca…",             ID_FILE_SAVEAS
```

File: base/applications/wordpad/lang/ro-RO.rc [hunk 2]
```diff
@@ -32,31 +32,31 @@ BEGIN
...
         MENUITEM "&Deschidere…\tCtrl+O",     ID_FILE_OPEN
-        MENUITEM "S&alvează\tCtrl+S",        ID_FILE_SAVE
-        MENUITEM "Salvea&ză ca…",            ID_FILE_SAVEAS
+        MENUITEM "S&alvare\tCtrl+S",         ID_FILE_SAVE
+        MENUITEM "Sal&vare ca…",             ID_FILE_SAVEAS
         MENUITEM SEPARATOR
         MENUITEM "I&mprimare…\tCtrl+P",      ID_PRINT
         MENUITEM "Pre&vizionare imprimare…", ID_PREVIEW
-        MENUITEM "&Configurare pagină…",     ID_PRINTSETUP
+        MENUITEM "&Inițializare pagină…",    ID_PRINTSETUP
```
Comment (JoachimHenze) [inline base/applications/wordpad/lang/ro-RO.rc:40]:
```suggestion
        MENUITEM "&Inițializare pagină…",     ID_PRINTSETUP
```

**Comment by AndreiMiloiu** [comment]
Any news?

**Pull request merged**